fhservices.cfg.j2

```diff
@@ -121,6 +121,7 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
+       # REVIEW: is this being applied to every pod/container? How does it work?
```

Comment:
I hope whatever we do is not relying on IP addresses:

> "Kubernetes applies IP addresses at the Pod scope - containers within a Pod share their network namespaces - including their IP address."

http://kubernetes.io/docs/admin/networking/#kubernetes-model


It seems that the disk check runs against the local file system, still puzzled how it is run for all containers / all disks.